### DISCUSSION TITLE
Cannot find module /app/dist/main

### DISCUSSION THREAD

**Discussion by SummerXXXX** [milvus-io/milvus-insight#160] [Q&A] [answered]
![Screenshot 2021-07-24 165837](https://user-images.githubusercontent.com/10771754/126863380-6228b958-ca78-46e9-9077-918c21aec6bb.png)

I got this "module not found error" while installing, is this because I do something wrong ?

**Comment by nameczz** [comment, marked as answer]
docker run -p 8000:3000 -e HOST_URL=http://127.0.0.1:8000 -e MILVUS_URL=127.0.0.1:19530 milvusdb/milvus-insight:latest

I tried this command in my computer. Everything works fine. 

Did you solve this problem now?

**Reply by SummerXXXX** [reply]
No, I still have this issue, I guess there is something in my computer😂

**Reply by shanghaikid** [reply]
what is your OS? windows 10? or linux?

**Reply by SummerXXXX** [reply]
centos7,  this is a public server shared by several people, so I guess maybe someone delete some files.

**Reply by shanghaikid** [reply]
Thanks, I think it's a bug. We will fix it and release a new one as soon as possible.

**Reply by shanghaikid** [reply]
Hi, we just release a new version, can you try it again?

**Reply by SummerXXXX** [reply]
thanks, this error is fixed, but I get another error while access http://127.0.0.1:8000:
![Screenshot 2021-07-26 181231](https://user-images.githubusercontent.com/10771754/126972650-dfc2d881-73cb-4e8a-84d0-65f2ca499716.png)
![Screenshot 2021-07-26 181128](https://user-images.githubusercontent.com/10771754/126972656-774df2c2-ea0f-46aa-a70b-a68ef1dff846.png)

**Reply by shanghaikid** [reply]
Sorry, my bad. please update the docker image again.

**Reply by SummerXXXX** [reply]
Yes, this is what I got after delete and re-download the docker image.
I just tried it again, still same error 'no such file or directory  /app/dist/build/index.html '

**Reply by nameczz** [reply]
> Yes, this is what I got after delete and re-download the docker image.
> I just tried it again, still same error 'no such file or directory /app/dist/build/index.html '

We released again and fix this error. I think it's working fine now.

**Reply by SummerXXXX** [reply]
I tried again, same error still happens 😥
![Screenshot 2021-07-26 223341](https://user-images.githubusercontent.com/10771754/127006824-e5741667-7d5a-4633-9f90-523d3bbfe194.png)

![Screenshot 2021-07-26 223157](https://user-images.githubusercontent.com/10771754/127006843-bb2c927c-51b3-4b54-bca3-5b8295ca4123.png)

if my sha256 is right ?

**Reply by nameczz** [reply]
Yes, you are right, Our CI has some problem, we will release it tomorrow and let you know. Thanks for your help!

**Reply by SummerXXXX** [reply]
Thanks for your hard working, very appreciate

**Reply by nameczz** [reply]
@SummerXXXX It's working fine now. Any more suggestions is welcome!
